packages/quantic/force-app/examples/main/lwc/exampleQuanticCategoryFacet/exampleQuanticCategoryFacet.js

```diff
@@ -0,0 +1,77 @@
+import {api, LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticFacet extends LightningElement {
+    @api engineId = 'quantic-category-facet-engine';
+    @track config = {};
+    isConfigured = false;
+
+    pageTitle = 'Quantic Category Facet';
+    pageDescription = 'The Quantic Category Facetis a list of values for a certain field occurring in the results, ordered using a configurable criterion.'
+    options = [
+        {
+            attribute: 'field',
+            label: 'Field',
+            description: 'The name of the field to display as a facet.',
+            defaultValue: 'geographicalhierarchy'
+        },
+        {
+            attribute: 'label',
+            label: 'Label',
+            description: 'The label to use as the facet title.',
+            defaultValue: 'Country'
+        },
+        {
+            attribute: 'basePath',
+            label: 'BasePath',
+            description: 'The base path shared by all facet values, separated by commas.',
+            defaultValue: ''
+        },
+        {
+            attribute: 'noFilterByBasePath',
+            label: 'No Filter By Base Path',
+            description: 'Whether not to use the `basePath` as a filter for the results.',
+            defaultValue: false
+        },
+        {
+            attribute: 'noFilterFacetCount',
+            label: 'No Filter Facet Count',
+            description: 'Whether not to exclude the parents of folded results when estimating the result count for each facet value.',
+            defaultValue: false
+        },
+        {
+            attribute: 'delimitingCharacter',
+            label: 'Delimiting Character',
+            description: 'The character that separates the values of the target multi-value field.',
```

Comment:
The actual description could be misleading. It depends on how the field has been configured in the index.

If the field is created as a **hierarchical field**, the default level delimiter is `;`, but mentioning `multi-value field` in the description is kind of misleading.

If the field is created as a **multi-value field**, the default multi-value delimiter is `;`, and the default level delimiter is `>`. In this case, the multi-value delimiter is handled internally by the platform, so it is transparent to the Search API consumers.

Bottom line, I think it would make things clearer to use "level delimiter" in the attribute description.